packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -18,6 +26,8 @@ export enum ValuePosition {
   Next = 1
 }
 
+const getClassNames = classNamesFunction<ISliderStyleProps, ISliderStyles>();
+@customizable('Label', ['theme', 'styles'])
```

Comment:
I thought we were no longer using this decorator? Styled is doing the same thing...